.github/workflows/gems_ci.yml

```diff
@@ -25,28 +29,67 @@ jobs:
 
       - uses: actions/checkout@v2
 
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+
+      - name: Set up JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ env.java_version }}
+
+      - name: Build and publish maven local smithy-ruby
+        run: |
+          git clone --depth 1 https://github.com/smithy-lang/smithy-ruby.git
+          cd smithy-ruby/codegen
+          ./gradlew clean build publishToMavenLocal
```

Comment:
Any way we can load rakefiles from smithy ruby instead? and have it run the build/publish rake tasks?

Reply:
Added task here: https://github.com/smithy-lang/smithy-ruby/commit/2cfc306c1dea68ceb8550cbffd7187613c3a5fda

And will use.